wespeaker/utils/executor.py

```diff
@@ -57,7 +57,6 @@ def run_epoch(dataloader, epoch_iter, model, criterion, optimizer, scheduler,
                 features = spec_aug(features,
                                     **configs['dataset_args']['spec_aug_args'])
 
-        with torch.cuda.amp.autocast(enabled=configs['enable_amp']):
```

Comment:
Thanks for contribution, but why removing this line here?

Reply:
same context manager is opened in line 50.
I'm not sure what's the purpose of 2 ctx managers if the same can be accomplished with a single use of `autocast` but with a broader scope.